### PR TITLE
Fix filter area in PivotGrid FiledPanel when there is a lot of fields on it (T645658)

### PR DESCRIPTION
**Pull request by sometimesElen**

### CODE REVIEW

File: styles/widgets/base/pivotGrid.less [hunk 1]
```diff
@@ -131,6 +131,30 @@
         }
     }
 
+    .dx-filter-header {
+        .dx-area-field-container{
```
Comment (vconst) [inline styles/widgets/base/pivotGrid.less]: The space is skipped

File: styles/widgets/base/pivotGrid.less [hunk 2]
```diff
@@ -131,6 +131,30 @@
...
+            border-spacing: 8px;
+            border-collapse: separate;
+        }
+
+        .dx-pivotgrid-fields-area-head {
+            td {
+                padding: 2px;
```
Comment (vconst) [inline styles/widgets/base/pivotGrid.less]: Use constant for number

File: styles/widgets/base/pivotGrid.less [hunk 3]
```diff
@@ -131,6 +131,30 @@
...
+            tr > td:first-child {
+                padding-left: 2px;
+            }
+        }
+    }
+
+    .dx-area-description-cell,
+    .dx-data-header {
+        tr > td:first-child {
+            padding-left: @PIVOTGRID_COMMON_LEFT_RIGHT_PADDING;
```
Comment (vconst) [inline styles/widgets/base/pivotGrid.less]: This style is not necessary

**Review by vconst — CHANGES_REQUESTED** [review]

File: styles/widgets/base/pivotGrid.less [hunk 4]
```diff
@@ -131,6 +133,23 @@
         }
     }
 
+    .dx-filter-header {
+        .dx-area-field-container {
+            border-spacing: @PIVOTGRID_FILTER_AREA_BORDING_SPACING;
```
Comment (vconst) [inline styles/widgets/base/pivotGrid.less]: BORDING_SPACING?

**Review by vconst — CHANGES_REQUESTED** [review]

**Review by roman-simionov — APPROVED** [review]

**Review by vconst — APPROVED** [review]

**Pull request merged**